pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [build-system]
-requires = ["poetry>=1.0.5"]
-build-backend = "poetry.masonry.api"
+requires = ["poetry-core>=1.1.1"]
```

Comment:
The latest version of [`poetry-core` ](https://pypi.org/project/poetry-core/#history)is `1.0.0`.
I guess it should be as in [the documentation](https://python-poetry.org/blog/announcing-poetry-1-1-0.html) that you already mentioned in the section **Standalone build backend**.

Reply:
Yes, you're definitely right. I'm sorry.  

https://github.com/python-poetry/poetry-core 

I'm surprised it worked with 1.1.1. without any warnings or errors.